subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/AbstractGradleExecuter.java

```diff
@@ -931,10 +947,13 @@ public GradleExecuter withBuildCacheEnabled() {
             int expectedDeprecationWarnings = AbstractGradleExecuter.this.expectedDeprecationWarnings;
             boolean expectStackTraces = !AbstractGradleExecuter.this.stackTraceChecksOn;
             boolean checkDeprecations = AbstractGradleExecuter.this.checkDeprecations;
+            boolean useRichConsole = AbstractGradleExecuter.this.useRichConsole;
 
             @Override
             public void execute(ExecutionResult executionResult) {
-                validate(executionResult.getNormalizedOutput(), "Standard output");
+                if (!useRichConsole) {
+                    validate(executionResult.getNormalizedOutput(), "Standard output");
```

Comment:
I reverted that back. I believe should now work with the validation but I still need to get the test working on Windows.